vimeo-networking/src/main/java/com/vimeo/networking/model/Recommendation.java

```diff
@@ -51,6 +51,35 @@
         USER
     }
 
+    // Needed for @UseStag
+    Recommendation() { }
+
+    public Recommendation(@Nullable Category category,
+                          @Nullable RecommendationType recommendationType,
+                          @Nullable User user,
+                          @Nullable Channel channel,
+                          @Nullable String description,
+                          @Nullable String resourceKey) {
+        mCategory = category;
+        if (recommendationType != null) {
+            switch (recommendationType) {
```

Comment:
It would help to clean up the code if this switch and getRecommendationType were localized in the the RecommendationType enum class.